lib/views/bottom-status.js

```diff
@@ -9,24 +9,50 @@ class BottomStatus extends HTMLElement{
     this.iconSpan.classList.add('icon')
     this.appendChild(this.iconSpan)
 
-    this.count = 0
+    this._linting = false
+    this._count = 0
```

Comment:
We tried to remove all the `_method / _var` to please write it `this.linting` etc...